### PR TITLE
Update ResizeHandler

### PR DESCRIPTION
**Pull request by edellucien**
# How to
Update ResizeHandler based on last changes from geops-react-ui

# Others

- [ ] It's not a hack or at least an unauthorized hack :).
- [ ] The images added are optimized.
- [ ] Everything in ticket description has been fixed.
- [ ] The author of the MR has made its own review before assigning the reviewer.
- [ ] IE11 tested.
- [ ] Labels applied. if it's a release? a hotfix?
- [ ] Tests added.

### CODE REVIEW

File: src/components/ResizeHandler/ResizeHandler.js [hunk 1]
```diff
@@ -15,6 +15,13 @@ const propTypes = {
   maxWidthBrkpts: PropTypes.objectOf(PropTypes.number),
   stylePropHeight: PropTypes.string,
   onResize: PropTypes.func,
+
+  // This property is used to re-apply the classes, for example when the className of the observed node changes.
+  forceUpdate: PropTypes.oneOfType([
+    PropTypes.string,
+    PropTypes.number,
+    PropTypes.bool,
+  ]),
```
Comment (oterral) [inline src/components/ResizeHandler/ResizeHandler.js:24]: Is that really needed ?

File: src/components/ResizeHandler/ResizeHandler.js [hunk 2]
```diff
@@ -99,26 +99,40 @@ class ResizeHandler extends PureComponent {
...
+    let newScreenHeight;
+
     for (let i = 0; i < entries.length; i += 1) {
       const entry = entries[i];
       const rect = entry.contentRect;
       const { height, width } = rect;
 
       if (maxWidthBrkpts) {
-        ResizeHandler.applyBreakpoints(entry, maxWidthBrkpts, width, 'w');
+        newScreenWidth = ResizeHandler.applyBreakpoints(
```
Comment (oterral) [inline src/components/ResizeHandler/ResizeHandler.js:116]: applyBreakpoints returns no value

File: src/components/ResizeHandler/ResizeHandler.js (hunk 1 above)
Comment (edellucien) [reply src/components/ResizeHandler/ResizeHandler.js:24]: I use it to update the IABP doc when changing disabled elements.

File: src/components/ResizeHandler/ResizeHandler.js (hunk 2 above)
Comment (edellucien) [reply src/components/ResizeHandler/ResizeHandler.js:119]: Indeed, I updated with the latest version from geops-react-ui

**Pull request merged**